torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -1979,7 +1954,24 @@ void ONNXShapeTypeInference(
     for (auto output : clone_node->outputs()) {
       n_graph->registerOutput(output);
     }
-
+    for (size_t i = 0; i < clone_node->outputs().size(); ++i) {
+      torch_to_onnx_output_name[n->output(i)->debugName()] = clone_node->output(i)->debugName();
```

Comment:
If my other comment is correct, the "torch" in this name would be inaccurate

Reply:
https://github.com/pytorch/pytorch/pull/75307#discussion_r878406888